codestodians/food-basket-api
# [Feature: #155059950] Update the ID fields in all models

#### What does this PR do?
- Updates the `ID` fields of the models.

#### Description of Task to be completed?
- Update the `ID` type for all models.
- Update schema to reflect model updates.
- Rerun migrations.
- Add fixtures (seed data) to aid in testing.
- Update test suites for all schema.

#### How should this be manually tested?
- Check out into the `ft-update-ID-field-type-in-models-155059950` branch.
- Update modules list by running `pip3 install -r requirements.txt`
- Run mutations, queries and tests as before.

#### Any background context you want to provide?
- This story arose as a result of the need to provide more security by virtue of making the respective `IDs` as random as possible.
- The sheer amount of files changed is due to the need to update the models, migrations, schema and tests.

#### What are the relevant pivotal tracker stories?
- [#155059950](https://www.pivotaltracker.com/story/show/155059950)
